### PR TITLE
Add support to test unsupported plugins

### PR DESCRIPTION
**Pull request by roaksoax**
Adds support to also test some unsupported plugins that are actively used by our community.

### CODE REVIEW

File: ci/test_plugins.rb [hunk 1]
```diff
@@ -140,6 +140,9 @@ def install_gem(gem_file)
     :filter => ["logstash-filter-aggregate", "logstash-filter-de_dot", "logstash-filter-throttle"],
     :output => ["logstash-output-csv", "logstash-output-graphite"]
   }
+  :unsupported => {
```
Comment (andsel) [inline ci/test_plugins.rb:143]: A comma is missed after the prev curly close

**Review by andsel — CHANGES_REQUESTED** [review]
I think there is a minimal syntax error

**Review by andsel — APPROVED** [review]
LGTM

**Review by dliappis — APPROVED** [review]
LGTM (as soon as https://buildkite.com/elastic/logstash-supported-plugins-test-pipeline/builds?branch=pull%2F15526%2Fmerge looks green)

**Pull request merged**